src/style.css

```diff
@@ -1,3 +1,126 @@
-body {
+:root {
   margin: 0;
+  padding: 0;
+
+  font-family: Roboto, Helvetica, sans-serif;
+  --secondary-color: #616070;
+  --blue-accent: #00acdc;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  margin: auto;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  font-size: 12px;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 16px 0;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+}
+
+.image {
+  margin-top: 32px;
+}
+
+.image--imac {
+  width: 160px;
+  height: 134px;
+}
+
+.product-name {
+  margin-bottom: 4px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+  margin-bottom: 24px;
+}
+
+.reviews__stars {
+  display: flex;
+}
+
+.stars {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) center no-repeat;
+}
+
+.stars:not(:last-child) {
+  margin-right: 4px;
+}
+
+.reviews__stars--4 :nth-child(-n + 4) {
+  background: url(images/star-active.svg) center no-repeat;
+}
+
+.reviews__name {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.price__text {
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.price__value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.button-buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid var(--blue-accent);
```

Comment:
Add default transparent border of the same width, and change its color on :hover